var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -3,752 +3,30 @@
-        tmp_path = env['TEST_TMPDIR']
-
-        # https://docs.bazel.build/versions/master/command-line-reference.html
-        args = [
-            # Don't allow user or system .bazelrc to override build settings
-            '--nohome_rc',
-            '--nosystem_rc',
-            # Bazel does not work properly on NFS, switch to /tmp
-            '--output_user_root=' + tmp_path,
-            'build',
-            # Spack logs don't handle colored output well
-            '--color=no',
-            '--jobs={0}'.format(make_jobs),
-            '--config=opt',
-            # Enable verbose output for failures
-            '--verbose_failures',
-            # Show (formatted) subcommands being executed
-            '--subcommands=pretty_print',
-            # Ask bazel to explain what it's up to
-            # Needs a filename as argument
-            '--explain=explainlogfile.txt',
-            '--incompatible_no_support_tools_in_action_inputs=false',
-            # Increase verbosity of explanation,
-            '--verbose_explanations',
-        ]
-
-        # See .bazelrc for when each config flag is supported
-        if spec.satisfies('@1.12.1:'):
-            if '+mkl' in spec:
-                args.append('--config=mkl')
-
-            if '+monolithic' in spec:
-                args.append('--config=monolithic')
-
-            if '+gdr' in spec:
-                args.append('--config=gdr')
-
-            if '+verbs' in spec:
-                args.append('--config=verbs')
-
-            if '+ngraph' in spec:
-                args.append('--config=ngraph')
-
-            if '+dynamic_kernels' in spec:
-                args.append('--config=dynamic_kernels')
-
-            if '+cuda' in spec:
-                args.append('--config=cuda')
-
-            if '~aws' in spec:
-                args.append('--config=noaws')
-
-            if '~gcp' in spec:
-                args.append('--config=nogcp')
-
-            if '~hdfs' in spec:
-                args.append('--config=nohdfs')
-
-            if '~nccl' in spec:
-                args.append('--config=nonccl')
-
-        if spec.satisfies('@1.12.1:2.0'):
-            if '~ignite' in spec:
-                args.append('--config=noignite')
-
-            if '~kafka' in spec:
-                args.append('--config=nokafka')
-
-        if spec.satisfies('@1.12.1,1.14:'):
-            if '+numa' in spec:
-                args.append('--config=numa')
-
-        if spec.satisfies('@2:'):
-            args.append('--config=v2')
-
-        args.append('//tensorflow/tools/pip_package:build_pip_package')
-
-        bazel(*args)
-
-        build_pip_package = Executable(
-            'bazel-bin/tensorflow/tools/pip_package/build_pip_package')
-        buildpath = join_path(self.stage.source_path, 'spack-build')
-        build_pip_package('--src', buildpath)
+    depends_on('py-numpy@1.16.0:',  type=('build', 'run'), when='@1.15:')
+    depends_on('cudnn')
+    depends_on('cuda')
```

Comment:
This list of dependencies seems a little short. I guess the installation via Pip pulls in a bunch of other stuff. Would be nice to have a longer list here, but in a pinch this should work.

Reply:
Based on our discussion, I added py-h5py.